### PR TITLE
ducs stinky code

### PR DESCRIPTION
**Pull request by GitAYYYer**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -64,6 +65,14 @@ client.on("messageCreate", async (message) => {
     message.channel.send(gnRegexMatch[0]);
   } else if (thanksRegexMatch) {
     message.channel.send("No worries <3");
+  } else {
+    const loveRegexMatch = loveRegex.exec(message.content);
+    if (loveRegexMatch) {
+      const lovedThing = loveRegexMatch[1];
+      message.channel.send(
+        `I love ${lovedThing}.\nI love ${lovedThing} Charlie!!!`
```
Comment (FR3D99) [inline index.js]:
```suggestion
        `I love ${lovedThing} Charlie\nI love ${lovedThing}!!!`
```

**Review by FR3D99 — APPROVED** [review]
lgtm (i have barely looked)

File: index.js (hunk 1 above)
Comment (GitAYYYer) [reply index.js]: no one asked

**Comment by GitAYYYer** [comment]
lgtm

**Pull request merged**